js/samples/07.whoBot/src/bot.ts

```diff
@@ -0,0 +1,269 @@
+import { ActivityTypes, Attachment, CardFactory, MemoryStorage, TurnContext } from 'botbuilder';
+import {
+    AI,
+    ActionPlanner,
+    ApplicationBuilder,
+    DefaultConversationState,
+    OpenAIModel,
+    PromptManager,
+    TurnState
+} from '@microsoft/teams-ai';
+import { configureUserAuthentication } from './userAuth';
+import { GraphClient } from './utils/graphClient';
+import { createCalendarEventCard, createUserEmailCard, createUserPersonalInformationCard } from './utils/cards';
+import { adapter } from '.';
+import * as path from 'path';
+
+if (!process.env.OPENAI_KEY && !process.env.AZURE_OPENAI_KEY) {
+    throw new Error('Missing environment variables - please check that OPENAI_KEY or AZURE_OPENAI_KEY is set.');
+}
+
+export const run = (context: TurnContext) => app.run(context);
+
+// Create AI components
+const model = new OpenAIModel({
+    // OpenAI Support
+    apiKey: process.env.OPENAI_KEY!,
```

Comment:
Should we comment the OpenAI settings? It seems developers only need to config one of OpenAI/AzureOpenAI, configuring both might be confusing. We can keep the commentted OpenAI settings as a reference for developers.

Reply:
We can keep both the settings so that the user can simply decide which one to use by setting the config value in the .env file. If they set `OPENAI_KEY` then open ai service will used. If they set `AZURE_OPENAI_KEY` and `AZURE_OPENAI_ENDPOINT` then azure open ai service will be used.